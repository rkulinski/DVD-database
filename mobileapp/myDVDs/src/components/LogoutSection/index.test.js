import React from 'react';
import { shallow } from 'enzyme';
import { LogoutSection } from './';


describe('Testing LogoutSection component', () => {
  it('renders as expected', () => {
    const wrapper = shallow(
      <LogoutSection onLogout={jest.fn()}/>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
