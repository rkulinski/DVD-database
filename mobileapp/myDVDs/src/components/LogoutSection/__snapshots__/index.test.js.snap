// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing LogoutSection component renders as expected 1`] = `
<Component
  style={
    Array [
      Object {
        "flex": 1,
        "height": 30,
        "marginBottom": 5,
      },
    ]
  }
>
  <Button
    onPress={[MockFunction]}
    style={
      Array [
        Object {
          "color": "black",
          "paddingTop": 5,
          "textAlign": "center",
        },
      ]
    }
    title="Logout (undefined)"
  />
</Component>
`;
